libs/events/src/CowEventEmitterProvider.ts

```diff
@@ -0,0 +1,4 @@
+import { createContext } from 'react'
```

Comment:
Would be great to make the lib framework-agnostic. I mean, people who would use the lib in their svelte/angular/etc apps wouldn't like to have React in their deps

Reply:
Not a blocker for now.
In the future we can make a new lib `@cowprotocol/events-react`

Reply:
Agree, and I thought about it, but I thought would be overkill. It will have the transitive dependency, but as long as they don't use the provider, it's all good I think

Reply:
Happy to change if this brings any issues

Reply:
This is not an issue any more as the events library don't depend on React context anymore in recent PRs